Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let tag = "0.1-alpha.1"
```

Comment:
side note: I don't think the `-alpha.1` suffix is supported by SwiftPM when we tag the release last I checked